shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -119,7 +120,20 @@ - (void)performCommonViewControllerInitialization {
   _orientationPreferences = UIInterfaceOrientationMaskAll;
   _statusBarStyle = UIStatusBarStyleDefault;
   _platformView =
-      std::make_shared<shell::PlatformViewIOS>(reinterpret_cast<CAEAGLLayer*>(self.view.layer));
+      std::make_shared<shell::PlatformViewIOS>(
+        reinterpret_cast<CAEAGLLayer*>(self.view.layer),
+        // First frame callback.
+        [self]() {
```

Comment:
I think you forgot to re-thread this call. `UIKit` calls may only be made on the platform thread. Also, you need a weak factory for thread safety.